addons/web/static/tests/views/list_tests.js

```diff
@@ -1687,24 +1687,34 @@ QUnit.module('Views', {
             View: ListView,
             model: 'foo',
             data: this.data,
-            arch: '<tree><field name="foo"/></tree>',
+            arch: '<tree>' +
+                    '<field name="foo"/>' +
+                    '<field name="bar" widget="toggle_button"/>' +
+                '</tree>',
         });
 
         // this is done to force the tooltip to show immediately instead of waiting
         // 1000 ms. not totally academic, but a short test suite is easier to sell :(
-        list.$('th:not(.o_list_record_selector)').tooltip('show', false);
+        list.$('th[data-name=foo]').tooltip('show', false);
 
-        list.$('th:not(.o_list_record_selector)').trigger($.Event('mouseenter'));
+        list.$('th[data-name=foo]').trigger($.Event('mouseenter'));
         assert.strictEqual($('.tooltip .oe_tooltip_string').length, 0, "should not have rendered a tooltip");
 
         config.debug = true;
         // it is necessary to rerender the list so tooltips can be properly created
         await list.reload();
-        list.$('th:not(.o_list_record_selector)').tooltip('show', false);
-
-        list.$('th:not(.o_list_record_selector)').trigger($.Event('mouseenter'));
+        list.$('th[data-name=foo]').tooltip('show', false);
+        list.$('th[data-name=foo]').trigger($.Event('mouseenter'));
         assert.strictEqual($('.tooltip .oe_tooltip_string').length, 1, "should have rendered a tooltip");
 
+        await list.reload();
+        list.$('th[data-name=bar]').tooltip('show', false);
+        list.$('th[data-name=bar]').trigger($.Event('mouseenter'));
+        assert.strictEqual($('.oe_tooltip_technical>li[data-item="widget"]').length,
```

Comment:
containsOnce?

Reply:
sure, a stupid copy paste :1st_place_medal:

Reply:
Good catch about ref, Github needs 7 digits apparently.
Nop, this has been introduced by a commit in master quite recently.